src/operation.ts

```diff
@@ -439,15 +433,22 @@ export default class Operation {
    * Convert the operation into an array of JSON Schema schemas for each available type of parameter available on the
    * operation.
    *
-   * @param globalDefaults Contains an object of user defined schema defaults.
-   */
-  getParametersAsJsonSchema(globalDefaults?: Record<string, unknown>) {
-    if (this.parameterJsonSchema) {
-      return this.parameterJsonSchema;
-    }
-
-    this.parameterJsonSchema = getParametersAsJsonSchema(this, this.api, globalDefaults);
-    return this.parameterJsonSchema;
+   * @param opts
+   * @param opts.globalDefaults Contains an object of user defined schema defaults.
+   * @param opts.mergeIntoBodyAndMetadata If you want the output to be two objects: body (contains
+   *    `body` and `formData` JSON Schema) and metadata (contains `path`, `query`, `cookie`, and
+   *    `header`).
+   * @param opts.retainDeprecatedProperties If you wish to **not** split out deprecated properties
```

Comment:
(insert same preface as above) what about "merge" instead of "retain"?

Reply:
This is similar. By default in the explorer we split out deprecated properties into a `deprecatedProps` object so we can show deprecated parameters separately from regular parameters and in `api` I don't want to do this because I'd prefer deprecated parameters to (eventually) have a `/** @deprecated */` docblock over them so when you try to reference it in an SDK your IDE lets you know.